simplefi-balancer-v2/src/vault.ts

```diff
@@ -0,0 +1,183 @@
+import { 
+  Address, 
+  BigInt, 
+  ethereum, 
+  log,
+} from "@graphprotocol/graph-ts"
+
+import {
+  PoolRegistered,
+  TokensDeregistered,
+  TokensRegistered,
+  PoolBalanceChanged,
+} from "../generated/Vault/Vault"
+
+import {
+  getOrCreateAccount,
+  getOrCreateERC20Token,
+  getOrCreateMarket,
+  updateMarket,
+  TokenBalance,
+} from "./common"
+
+import {
+  Pool as PoolEntity,
+  PoolId as PoolIdEntity,
+  Token as TokenEntity,
+  Mint as MintEntity,
+  Burn as BurnEntity,
+  Market as MarketEntity,
+} from "../generated/schema"
+
+import { 
+  ProtocolName, 
+  ProtocolType,
+  PoolSpecialization
+} from "./constants"
+
+import {
+  createOrUpdatePositionOnMint,
+  createOrUpdatePositionOnBurn
+} from "./market"
+
+export function handlePoolRegistered(event: PoolRegistered): void {
+  let pool = new PoolEntity(event.params.poolId.toHexString())
+  pool.address = event.params.poolAddress.toHexString();
+
+  let evmEvent = event as ethereum.Event
```

Comment:
@NelsonGaldeman You don't need to create this local variable `evmEvent` here. You can directly do `event.block.number` and `event.block.timestamp` in lines below.

Reply:
Hmm I remember my IDE was complaining there but now it's not maybe something changed or I had a bad import when I was writing it!